astropy/stats/funcs.py

```diff
@@ -444,6 +446,193 @@ def true_efficiency(x):
+    distribution with the indicated number of degrees of freedom and
+    :math:\alpha is the one-tailed probability of the normal
+    distribution (at the point given by the parameter 'sigma'). See
+    [Maxwell 2011][maxw11] for further details.
+
+    Examples
+    --------
+
+    >>> poisson_conf_interval(np.arange(10), interval='root-n').T
+    array([[  0.        ,   0.        ],
+           [  0.        ,   2.        ],
+           [  0.58578644,   3.41421356],
+           [  1.26794919,   4.73205081],
+           [  2.        ,   6.        ],
+           [  2.76393202,   7.23606798],
+           [  3.55051026,   8.44948974],
+           [  4.35424869,   9.64575131],
+           [  5.17157288,  10.82842712],
+           [  6.        ,  12.        ]])
+
+    >>> poisson_conf_interval(np.arange(10), interval='root-n-0').T
+    array([[  0.        ,   1.        ],
+           [  0.        ,   2.        ],
+           [  0.58578644,   3.41421356],
+           [  1.26794919,   4.73205081],
+           [  2.        ,   6.        ],
+           [  2.76393202,   7.23606798],
+           [  3.55051026,   8.44948974],
+           [  4.35424869,   9.64575131],
+           [  5.17157288,  10.82842712],
+           [  6.        ,  12.        ]])
+
+    >>> poisson_conf_interval(np.arange(10), interval='pearson').T
+    array([[  0.        ,   1.        ],
+           [  0.38196601,   2.61803399],
+           [  1.        ,   4.        ],
+           [  1.69722436,   5.30277564],
+           [  2.43844719,   6.56155281],
+           [  3.20871215,   7.79128785],
+           [  4.        ,   9.        ],
+           [  4.8074176 ,  10.1925824 ],
+           [  5.62771868,  11.37228132],
+           [  6.45861873,  12.54138127]])
+    
+    >>> poisson_conf_interval(np.arange(10),
+    ...                       interval='frequentist-confidence').T
+    array([[  0.        ,   1.84102165],
+           [  0.17275378,   3.29952656],
+           [  0.70818544,   4.63785962],
+           [  1.36729531,   5.91818583],
+           [  2.08566081,   7.16275317],
+           [  2.84030886,   8.38247265],
+           [  3.62006862,   9.58364155],
+           [  4.41852954,  10.77028072],
+           [  5.23161394,  11.94514152],
+           [  6.05653896,  13.11020414]])
+
+    >>> poisson_conf_interval(7,
+    ...                       interval='frequentist-confidence').T
+    array([  4.41852954,  10.77028072])
+                   
+    [pois_eb]: http://www-cdf.fnal.gov/physics/statistics/notes/pois_eb.txt
+    [ErrorBars]: http://www.pp.rhul.ac.uk/~cowan/atlas/ErrorBars.pdf
+    [ac12]: http://adsabs.harvard.edu/abs/2012EPJP..127...24A
+    [maxw11]: http://adsabs.harvard.edu/abs/2011arXiv1102.0822M
+    [gehrels86]: http://adsabs.harvard.edu/abs/1986ApJ...303..336G
+    
+    """
+
+    if interval == 'root-n':
+        if sigma!=1:
+            raise ValueError("Only sigma=1 supported for interval %s" % interval)
```

Comment:
Nitpick – the [Astropy coding guidelines](http://astropy.readthedocs.org/en/latest/development/codeguide.html) say that
> The new Python 3 formatting style should be used (i.e. "{0:s}".format("spam")